### PR TITLE
Feature/form updates

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

File: app/models/concerns/scholars_archive/article_metadata.rb [hunk 1]
```diff
@@ -32,6 +33,50 @@ module ArticleMetadata
       property :web_of_science_uid, predicate: ::RDF::URI.new('http://opaquenamespace.org/ns/webOfScienceUid'), multiple: false do |index|
         index.as :stored_searchable
       end
+
+      define_singleton_method :article_properties do
```
Comment (CGillen) [inline app/models/concerns/scholars_archive/article_metadata.rb]: Do you have plans for these methods? They don't seem used in this PR and didn't exist before as far as I can tell.

Comment (straleyb) [reply app/models/concerns/scholars_archive/article_metadata.rb]: Yeah, I wanna get these initial refactors in and get it all playing nicely. This method would replace the model::TERMS which should lower the amount of lists we carry around smaller.

File: config/locales/hyrax.en.yml [hunk 2]
```diff
@@ -85,7 +85,7 @@ en:
 
     works:
       create:
-        after_create_html: "ScholarsArchive@OSU has received your upload. Your file is currently being processed. Thank you for your deposit."
+        after_create_html: "ScholarsArchive@OSU has received your deposit. Your file is currently being processed; some fields may take a few minutes to display."
```
[the remark this answers is not in the document]
Comment (straleyb) [reply config/locales/hyrax.en.yml]: Yeah..... Its git funny business.

**Comment by straleyb** [comment]
@CGillen Yeah, I can do that. Ill move everything into their own modules. I think thats the ultimate goal here.

**Pull request merged**